src/vstest.console/Processors/ListExtensionsArgumentProcessor.cs

```diff
@@ -0,0 +1,236 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
Are these getting deprecated ? If yes, should we let the user know that these options are getting deprecated via message.